Src/DependencyCollector/Net46.Tests/DependencyTrackingTelemetryModuleTestNet46.cs

```diff
@@ -0,0 +1,136 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector
```

Comment:
specific tests for Http Reflection hook implemented in DiagnosticSource and used in DependencyCollector